shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/AboutAppScreen.kt

```diff
@@ -0,0 +1,89 @@
+package org.jetbrains.kotlinconf.screens
+
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import kotlinconfapp.shared.generated.resources.Res
+import kotlinconfapp.shared.generated.resources.about_app_description
+import kotlinconfapp.shared.generated.resources.about_app_link_github
+import kotlinconfapp.shared.generated.resources.about_app_link_privacy_policy
+import kotlinconfapp.shared.generated.resources.about_app_link_rate
+import kotlinconfapp.shared.generated.resources.about_app_link_settings
+import kotlinconfapp.shared.generated.resources.about_app_link_terms_of_use
+import kotlinconfapp.shared.generated.resources.about_app_title
+import kotlinconfapp.shared.generated.resources.arrow_left_24
+import kotlinconfapp.shared.generated.resources.arrow_up_right_24
+import kotlinconfapp.shared.generated.resources.navigate_back
+import org.jetbrains.compose.resources.stringResource
+import org.jetbrains.kotlinconf.ui.components.Divider
+import org.jetbrains.kotlinconf.ui.components.MainHeaderTitleBar
+import org.jetbrains.kotlinconf.ui.components.PageMenuItem
+import org.jetbrains.kotlinconf.ui.components.StyledText
+import org.jetbrains.kotlinconf.ui.components.TopMenuButton
+import org.jetbrains.kotlinconf.ui.theme.KotlinConfTheme
+import org.jetbrains.kotlinconf.utils.getStoreUrl
+
+@Composable
+fun AboutAppScreen(
+    onBack: () -> Unit,
+    onGitHubRepo: () -> Unit,
+    onRateApp: () -> Unit,
+    onSettings: () -> Unit,
+    onPrivacyPolicy: () -> Unit,
+    onTermsOfUse: () -> Unit,
+) {
+    Column(Modifier.fillMaxSize()) {
+        MainHeaderTitleBar(
+            title = stringResource(Res.string.about_app_title),
+            startContent = {
+                TopMenuButton(
+                    icon = Res.drawable.arrow_left_24,
+                    contentDescription = stringResource(Res.string.navigate_back),
+                    onClick = onBack,
+                )
+            }
+        )
+        Divider(1.dp, KotlinConfTheme.colors.strokePale)
+        Column(
+            Modifier
+                .verticalScroll(rememberScrollState())
+                .fillMaxSize()
+                .padding(horizontal = 12.dp)
+                .padding(bottom = 12.dp),
+            verticalArrangement = Arrangement.spacedBy(8.dp)
```

Comment:
I've added a `ScreenWithTitle` to hide this boilerplate in my PR, we may reuse it later

Reply:
Great, started using this for the About screen!